sdcm/utils/health_checker.py

```diff
@@ -16,7 +16,9 @@ def check_nodes_status(nodes_status, current_node):
     for node_ip, node_properties in nodes_status.items():
         if node_properties['status'] != "UN":
             is_target = current_node.print_node_running_nemesis(node_ip)
-            ClusterHealthValidatorEvent(type='critical', name='NodeStatus', status=Severity.CRITICAL,
+            # FIXME: change status from Severity.CRITICAL because of issue number
+            #  https://github.com/scylladb/scylla-enterprise/issues/1419 and must be reverted once it is fixed.
+            ClusterHealthValidatorEvent(type='critical', name='NodeStatus', status=Severity.ERROR,
```

Comment:
This is a problem... it's too general.
We need this check on the general case, but ignore it only when this node is replaced.
The question is if it's possible doing it....

Reply:
i'm working on a more robust workaround, as we chatted now.